instance/models/mixins/openedx_database.py

```diff
@@ -182,14 +183,38 @@ def _get_mysql_database_name(self, suffix):
         assert len(mysql_database_name) <= 64
         return mysql_database_name
 
+    def create_db(self, db_suffix):
+        """
+        The database is normally created via ansible playbooks. However if you need to create the database manually
+        for some reason, such as after dropping it in anticipation of restoring a backup or fixing a failed initial
+        provision, this method handles it for you.
+        """
+        assert db_suffix
+        db_name = self._get_mysql_database_name(db_suffix)
+        with self.mysql_server.get_admin_cursor() as cursor:
+            # We can't use prepared style queries here, because the database name can't be a string.
+            # We should never use this with user input anyway, so it should be OK.
+            cursor.execute(f'CREATE DATABASE {db_name}')
```

Comment:
@Kelketek, @nizarmah, this will break things because the statement doesn't specify a character set like it is done [here](https://github.com/open-craft/opencraft/blob/master/instance/models/mixins/database.py#L94) and the default character set on the server is `latin1` and not `utf-8`. Mixing character sets within a database can cause issues.

Reply:
😯  @lgp171188 thanks for noticing that! I'll create a ticket for it tomorrow, I'll add it to Stretch Goals so a firefighter or maybe a newcomer can address this.

Reply:
@lgp171188 I just wanted to let you know that [a ticket has been created](https://tasks.opencraft.com/browse/SE-4282) by @Arti3DPlayer for this 👍🏼